### PR TITLE
Fix small format issue in the "Install Kedro" docs

### PR DESCRIPTION
**Pull request by laisbsc**
## Description
Fix small formatting issues on the 'Getting Started' page of the docs.

## Checklist

- [x] Read the [contributing](https://github.com/quantumblacklabs/kedro/blob/master/CONTRIBUTING.md) guidelines
- [x] Updated the documentation to reflect the code changes
- [ ] Added a description of this change and added my name to the list of supporting contributions in the [`RELEASE.md`](https://github.com/quantumblacklabs/kedro/blob/master/RELEASE.md) file
- [ ] Added tests to cover my changes

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: docs/source/02_get_started/03_hello_kedro.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ You can copy the example as one chunk of code from the bottom of this page.
 
 > Note: We do not create a Kedro project in this first example, but illustrate the concepts within a single `.py` file.
 
-> Note: We do not create a Kedro project in this first example, but illustrate the concepts within a single `.py` file.
```
Comment (deepyaman) [inline docs/source/02_get_started/03_hello_kedro.md:9]: Also fixed in #648.

**Review by deepyaman — APPROVED** [review]
@laisbsc ❤️

**Review by MerelTheisenQB — APPROVED** [review]
Thanks @laisbsc  🥇 ❤️

**Review by lorenabalan — APPROVED** [review]
Nice work! Just some conflicts to be resolved before we merge this. 
Good to see you back around here! 😊

**Comment by laisbsc** [comment]
> Nice work! Just some conflicts to be resolved before we merge this.

Fixed!

> Good to see you back around here! 😊 

It is my pleasure! Miss ya all ❤️  And congratulations on the release!

Hope the holidays were fun.

File: docs/source/02_get_started/03_hello_kedro.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ You can copy the example as one chunk of code from the bottom of this page.
 
 > Note: We do not create a Kedro project in this first example, but illustrate the concepts within a single `.py` file.
 
+
```
Comment (lorenabalan) [inline docs/source/02_get_started/03_hello_kedro.md:9]:
```suggestion
```

**Pull request merged**